howtos/pubsub/how_to_use_local_transactions.py

```diff
@@ -0,0 +1,153 @@
+        transactional_service = service.create_transactional_service_builder().build().connect()
+        # Start a receiver for every queue
+        for queue in queues:
+            receiver = transactional_service.create_transactional_message_receiver_builder().build(queue).start()
+            receivers.append(receiver)
+
+        for _ in range(count):
+            # Grab a message from each receiver/queue
+            messages_in_this_transaction = []
+            for receiver in receivers:
+                message = receiver.receive_message()
+                if message is None:
+                    if messages_in_this_transaction:
+                        # One of the queues ran empty, rolling back the partial transaction:
+                        transactional_service.rollback()
+                    break
+                messages_in_this_transaction.append(message)
+            else: # no break
+                try:
+                    # Consume a message from all queues, or from none:
+                    transactional_service.commit()
+                    messages.extend(messages_in_this_transaction)
+                except TransactionRollbackError:
+                    # Commit failed.
+                    pass
+                except UnknownTransactionStateError:
+                    # Commit outcome unknown. This should be very rare.
+                    pass
+        transactional_service.disconnect()
+        return messages
+
+
+    @staticmethod
+    def run():
+        """ Execute the demonstrator methods. """
+        try:
+            number_of_queues = 5
+            messages_per_queue = 3
+            messaging_service = MessagingService.builder().from_properties(boot.broker_properties()).build()
+            messaging_service.connect()
+            print(f'Message service is connected? {messaging_service.is_connected}')
+            topics = []
+            queues = []
+            for i in range(number_of_queues):
+                topic_name = constants.TOPIC_ENDPOINT_DEFAULT + str(i)
+                topic = Topic.of(topic_name)
+                topics.append(topic)
+                queue_name = constants.QUEUE_NAME_FORMAT.substitute(iteration=topic_name)
+                semp.create_queue(name=queue_name,
+                                  msg_vpn_name=broker_props[service_properties.VPN_NAME],
+                                  access_type="exclusive",
+                                  egress_enabled=True)
+                semp.add_topic_to_queue(topic_name=topic_name,
+                                        queue_name=queue_name,
+                                        msg_vpn_name=broker_props[service_properties.VPN_NAME])
+                queue = Queue.durable_exclusive_queue(queue_name)
+                queues.append(queue)
+
+                # Publish the expected number of messages to the queues
+                HowtoForLocalTransactions.publish_to_topics(messaging_service, topics, messages_per_queue)
```

Comment:
For the samples sake, can we assign this to a `messages_delivered_to_topics` var and just print it

Reply:
Done.
I'm also printing out the received messages while at it.